WARNING: $PATH does not agree with $PATH_modshare counter. The following directories' usage counters were adjusted to match. Note that this may mean that module unloading may not work correctly.
 /opt/slurm/bin /sw/arch/Debian10/EB_production/2020/software/CUDAcore/11.0.2/nvvm/bin /hpc/eb/compilerwrappers/linkers /sw/arch/Debian10/EB_production/2020/software/binutils/2.34-GCCcore-9.3.0/bin /sw/arch/Debian10/EB_production/2020/software/CUDAcore/11.0.2 /sw/arch/Debian10/EB_production/2020/software/GCCcore/9.3.0/bin /sw/arch/Debian10/EB_production/2020/software/CUDAcore/11.0.2/bin /opt/slurm/sbin /hpc/eb/compilerwrappers/compilers
2021-02-14 21:38:40.997280: W tensorflow/stream_executor/platform/default/dso_loader.cc:59] Could not load dynamic library 'libcudart.so.10.1'; dlerror: libcudart.so.10.1: cannot open shared object file: No such file or directory; LD_LIBRARY_PATH: /sw/arch/Debian10/EB_production/2020/software/OpenMPI/4.0.3-GCC-9.3.0/lib:/sw/arch/Debian10/EB_production/2020/software/OpenMPI/4.0.3-GCC-9.3.0/lib:/sw/arch/Debian10/EB_production/2020/software/Python/3.8.2-GCCcore-9.3.0/lib:/sw/arch/Debian10/EB_production/2020/software/libffi/3.3-GCCcore-9.3.0/lib64:/sw/arch/Debian10/EB_production/2020/software/libffi/3.3-GCCcore-9.3.0/lib:/sw/arch/Debian10/EB_production/2020/software/GMP/6.2.0-GCCcore-9.3.0/lib:/sw/arch/Debian10/EB_production/2020/software/SQLite/3.31.1-GCCcore-9.3.0/lib:/sw/arch/Debian10/EB_production/2020/software/Tcl/8.6.10-GCCcore-9.3.0/lib:/sw/arch/Debian10/EB_production/2020/software/libreadline/8.0-GCCcore-9.3.0/lib:/sw/arch/Debian10/EB_production/2020/software/ncurses/6.2-GCCcore-9.3.0/lib:/sw/arch/Debian10/EB_production/2020/software/bzip2/1.0.8-GCCcore-9.3.0/lib:/sw/arch/Debian10/EB_production/2020/software/OpenMPI/4.0.3-GCC-9.3.0/lib:/sw/arch/Debian10/EB_production/2020/software/UCX/1.8.0-GCCcore-9.3.0/lib:/sw/arch/Debian10/EB_production/2020/software/hwloc/2.2.0-GCCcore-9.3.0/lib:/sw/arch/Debian10/EB_production/2020/software/libpciaccess/0.16-GCCcore-9.3.0/lib:/sw/arch/Debian10/EB_production/2020/software/libxml2/2.9.10-GCCcore-9.3.0/lib:/sw/arch/Debian10/EB_production/2020/software/XZ/5.2.5-GCCcore-9.3.0/lib:/sw/arch/Debian10/EB_production/2020/software/numactl/2.0.13-GCCcore-9.3.0/lib:/sw/arch/Debian10/EB_production/2020/software/binutils/2.34-GCCcore-9.3.0/lib:/sw/arch/Debian10/EB_production/2020/software/zlib/1.2.11-GCCcore-9.3.0/lib:/sw/arch/Debian10/EB_production/2020/software/GCCcore/9.3.0/lib64:/sw/arch/Debian10/EB_production/2020/software/GCCcore/9.3.0/lib
2021-02-14 21:38:40.997311: I tensorflow/stream_executor/cuda/cudart_stub.cc:29] Ignore above cudart dlerror if you do not have a GPU set up on your machine.
2021-02-14 21:38:40.997279: W tensorflow/stream_executor/platform/default/dso_loader.cc:59] Could not load dynamic library 'libcudart.so.10.1'; dlerror: libcudart.so.10.1: cannot open shared object file: No such file or directory; LD_LIBRARY_PATH: /sw/arch/Debian10/EB_production/2020/software/OpenMPI/4.0.3-GCC-9.3.0/lib:/sw/arch/Debian10/EB_production/2020/software/OpenMPI/4.0.3-GCC-9.3.0/lib:/sw/arch/Debian10/EB_production/2020/software/Python/3.8.2-GCCcore-9.3.0/lib:/sw/arch/Debian10/EB_production/2020/software/libffi/3.3-GCCcore-9.3.0/lib64:/sw/arch/Debian10/EB_production/2020/software/libffi/3.3-GCCcore-9.3.0/lib:/sw/arch/Debian10/EB_production/2020/software/GMP/6.2.0-GCCcore-9.3.0/lib:/sw/arch/Debian10/EB_production/2020/software/SQLite/3.31.1-GCCcore-9.3.0/lib:/sw/arch/Debian10/EB_production/2020/software/Tcl/8.6.10-GCCcore-9.3.0/lib:/sw/arch/Debian10/EB_production/2020/software/libreadline/8.0-GCCcore-9.3.0/lib:/sw/arch/Debian10/EB_production/2020/software/ncurses/6.2-GCCcore-9.3.0/lib:/sw/arch/Debian10/EB_production/2020/software/bzip2/1.0.8-GCCcore-9.3.0/lib:/sw/arch/Debian10/EB_production/2020/software/OpenMPI/4.0.3-GCC-9.3.0/lib:/sw/arch/Debian10/EB_production/2020/software/UCX/1.8.0-GCCcore-9.3.0/lib:/sw/arch/Debian10/EB_production/2020/software/hwloc/2.2.0-GCCcore-9.3.0/lib:/sw/arch/Debian10/EB_production/2020/software/libpciaccess/0.16-GCCcore-9.3.0/lib:/sw/arch/Debian10/EB_production/2020/software/libxml2/2.9.10-GCCcore-9.3.0/lib:/sw/arch/Debian10/EB_production/2020/software/XZ/5.2.5-GCCcore-9.3.0/lib:/sw/arch/Debian10/EB_production/2020/software/numactl/2.0.13-GCCcore-9.3.0/lib:/sw/arch/Debian10/EB_production/2020/software/binutils/2.34-GCCcore-9.3.0/lib:/sw/arch/Debian10/EB_production/2020/software/zlib/1.2.11-GCCcore-9.3.0/lib:/sw/arch/Debian10/EB_production/2020/software/GCCcore/9.3.0/lib64:/sw/arch/Debian10/EB_production/2020/software/GCCcore/9.3.0/lib
2021-02-14 21:38:40.997311: I tensorflow/stream_executor/cuda/cudart_stub.cc:29] Ignore above cudart dlerror if you do not have a GPU set up on your machine.
2021-02-14 21:38:40.997280: W tensorflow/stream_executor/platform/default/dso_loader.cc:59] Could not load dynamic library 'libcudart.so.10.1'; dlerror: libcudart.so.10.1: cannot open shared object file: No such file or directory; LD_LIBRARY_PATH: /sw/arch/Debian10/EB_production/2020/software/OpenMPI/4.0.3-GCC-9.3.0/lib:/sw/arch/Debian10/EB_production/2020/software/OpenMPI/4.0.3-GCC-9.3.0/lib:/sw/arch/Debian10/EB_production/2020/software/Python/3.8.2-GCCcore-9.3.0/lib:/sw/arch/Debian10/EB_production/2020/software/libffi/3.3-GCCcore-9.3.0/lib64:/sw/arch/Debian10/EB_production/2020/software/libffi/3.3-GCCcore-9.3.0/lib:/sw/arch/Debian10/EB_production/2020/software/GMP/6.2.0-GCCcore-9.3.0/lib:/sw/arch/Debian10/EB_production/2020/software/SQLite/3.31.1-GCCcore-9.3.0/lib:/sw/arch/Debian10/EB_production/2020/software/Tcl/8.6.10-GCCcore-9.3.0/lib:/sw/arch/Debian10/EB_production/2020/software/libreadline/8.0-GCCcore-9.3.0/lib:/sw/arch/Debian10/EB_production/2020/software/ncurses/6.2-GCCcore-9.3.0/lib:/sw/arch/Debian10/EB_production/2020/software/bzip2/1.0.8-GCCcore-9.3.0/lib:/sw/arch/Debian10/EB_production/2020/software/OpenMPI/4.0.3-GCC-9.3.0/lib:/sw/arch/Debian10/EB_production/2020/software/UCX/1.8.0-GCCcore-9.3.0/lib:/sw/arch/Debian10/EB_production/2020/software/hwloc/2.2.0-GCCcore-9.3.0/lib:/sw/arch/Debian10/EB_production/2020/software/libpciaccess/0.16-GCCcore-9.3.0/lib:/sw/arch/Debian10/EB_production/2020/software/libxml2/2.9.10-GCCcore-9.3.0/lib:/sw/arch/Debian10/EB_production/2020/software/XZ/5.2.5-GCCcore-9.3.0/lib:/sw/arch/Debian10/EB_production/2020/software/numactl/2.0.13-GCCcore-9.3.0/lib:/sw/arch/Debian10/EB_production/2020/software/binutils/2.34-GCCcore-9.3.0/lib:/sw/arch/Debian10/EB_production/2020/software/zlib/1.2.11-GCCcore-9.3.0/lib:/sw/arch/Debian10/EB_production/2020/software/GCCcore/9.3.0/lib64:/sw/arch/Debian10/EB_production/2020/software/GCCcore/9.3.0/lib
2021-02-14 21:38:40.997311: I tensorflow/stream_executor/cuda/cudart_stub.cc:29] Ignore above cudart dlerror if you do not have a GPU set up on your machine.
2021-02-14 21:38:40.997280: W tensorflow/stream_executor/platform/default/dso_loader.cc:59] Could not load dynamic library 'libcudart.so.10.1'; dlerror: libcudart.so.10.1: cannot open shared object file: No such file or directory; LD_LIBRARY_PATH: /sw/arch/Debian10/EB_production/2020/software/OpenMPI/4.0.3-GCC-9.3.0/lib:/sw/arch/Debian10/EB_production/2020/software/OpenMPI/4.0.3-GCC-9.3.0/lib:/sw/arch/Debian10/EB_production/2020/software/Python/3.8.2-GCCcore-9.3.0/lib:/sw/arch/Debian10/EB_production/2020/software/libffi/3.3-GCCcore-9.3.0/lib64:/sw/arch/Debian10/EB_production/2020/software/libffi/3.3-GCCcore-9.3.0/lib:/sw/arch/Debian10/EB_production/2020/software/GMP/6.2.0-GCCcore-9.3.0/lib:/sw/arch/Debian10/EB_production/2020/software/SQLite/3.31.1-GCCcore-9.3.0/lib:/sw/arch/Debian10/EB_production/2020/software/Tcl/8.6.10-GCCcore-9.3.0/lib:/sw/arch/Debian10/EB_production/2020/software/libreadline/8.0-GCCcore-9.3.0/lib:/sw/arch/Debian10/EB_production/2020/software/ncurses/6.2-GCCcore-9.3.0/lib:/sw/arch/Debian10/EB_production/2020/software/bzip2/1.0.8-GCCcore-9.3.0/lib:/sw/arch/Debian10/EB_production/2020/software/OpenMPI/4.0.3-GCC-9.3.0/lib:/sw/arch/Debian10/EB_production/2020/software/UCX/1.8.0-GCCcore-9.3.0/lib:/sw/arch/Debian10/EB_production/2020/software/hwloc/2.2.0-GCCcore-9.3.0/lib:/sw/arch/Debian10/EB_production/2020/software/libpciaccess/0.16-GCCcore-9.3.0/lib:/sw/arch/Debian10/EB_production/2020/software/libxml2/2.9.10-GCCcore-9.3.0/lib:/sw/arch/Debian10/EB_production/2020/software/XZ/5.2.5-GCCcore-9.3.0/lib:/sw/arch/Debian10/EB_production/2020/software/numactl/2.0.13-GCCcore-9.3.0/lib:/sw/arch/Debian10/EB_production/2020/software/binutils/2.34-GCCcore-9.3.0/lib:/sw/arch/Debian10/EB_production/2020/software/zlib/1.2.11-GCCcore-9.3.0/lib:/sw/arch/Debian10/EB_production/2020/software/GCCcore/9.3.0/lib64:/sw/arch/Debian10/EB_production/2020/software/GCCcore/9.3.0/lib
2021-02-14 21:38:40.997311: I tensorflow/stream_executor/cuda/cudart_stub.cc:29] Ignore above cudart dlerror if you do not have a GPU set up on your machine.
2021-02-14 21:38:44.912034: W tensorflow/stream_executor/platform/default/dso_loader.cc:59] Could not load dynamic library 'libcuda.so.1'; dlerror: libcuda.so.1: cannot open shared object file: No such file or directory; LD_LIBRARY_PATH: /sw/arch/Debian10/EB_production/2020/software/OpenMPI/4.0.3-GCC-9.3.0/lib:/sw/arch/Debian10/EB_production/2020/software/OpenMPI/4.0.3-GCC-9.3.0/lib:/sw/arch/Debian10/EB_production/2020/software/Python/3.8.2-GCCcore-9.3.0/lib:/sw/arch/Debian10/EB_production/2020/software/libffi/3.3-GCCcore-9.3.0/lib64:/sw/arch/Debian10/EB_production/2020/software/libffi/3.3-GCCcore-9.3.0/lib:/sw/arch/Debian10/EB_production/2020/software/GMP/6.2.0-GCCcore-9.3.0/lib:/sw/arch/Debian10/EB_production/2020/software/SQLite/3.31.1-GCCcore-9.3.0/lib:/sw/arch/Debian10/EB_production/2020/software/Tcl/8.6.10-GCCcore-9.3.0/lib:/sw/arch/Debian10/EB_production/2020/software/libreadline/8.0-GCCcore-9.3.0/lib:/sw/arch/Debian10/EB_production/2020/software/ncurses/6.2-GCCcore-9.3.0/lib:/sw/arch/Debian10/EB_production/2020/software/bzip2/1.0.8-GCCcore-9.3.0/lib:/sw/arch/Debian10/EB_production/2020/software/OpenMPI/4.0.3-GCC-9.3.0/lib:/sw/arch/Debian10/EB_production/2020/software/UCX/1.8.0-GCCcore-9.3.0/lib:/sw/arch/Debian10/EB_production/2020/software/hwloc/2.2.0-GCCcore-9.3.0/lib:/sw/arch/Debian10/EB_production/2020/software/libpciaccess/0.16-GCCcore-9.3.0/lib:/sw/arch/Debian10/EB_production/2020/software/libxml2/2.9.10-GCCcore-9.3.0/lib:/sw/arch/Debian10/EB_production/2020/software/XZ/5.2.5-GCCcore-9.3.0/lib:/sw/arch/Debian10/EB_production/2020/software/numactl/2.0.13-GCCcore-9.3.0/lib:/sw/arch/Debian10/EB_production/2020/software/binutils/2.34-GCCcore-9.3.0/lib:/sw/arch/Debian10/EB_production/2020/software/zlib/1.2.11-GCCcore-9.3.0/lib:/sw/arch/Debian10/EB_production/2020/software/GCCcore/9.3.0/lib64:/sw/arch/Debian10/EB_production/2020/software/GCCcore/9.3.0/lib
2021-02-14 21:38:44.912066: W tensorflow/stream_executor/cuda/cuda_driver.cc:312] failed call to cuInit: UNKNOWN ERROR (303)
2021-02-14 21:38:44.912080: I tensorflow/stream_executor/cuda/cuda_diagnostics.cc:156] kernel driver does not appear to be running on this host (r37n1.lisa.surfsara.nl): /proc/driver/nvidia/version does not exist
2021-02-14 21:38:44.912037: W tensorflow/stream_executor/platform/default/dso_loader.cc:59] Could not load dynamic library 'libcuda.so.1'; dlerror: libcuda.so.1: cannot open shared object file: No such file or directory; LD_LIBRARY_PATH: /sw/arch/Debian10/EB_production/2020/software/OpenMPI/4.0.3-GCC-9.3.0/lib:/sw/arch/Debian10/EB_production/2020/software/OpenMPI/4.0.3-GCC-9.3.0/lib:/sw/arch/Debian10/EB_production/2020/software/Python/3.8.2-GCCcore-9.3.0/lib:/sw/arch/Debian10/EB_production/2020/software/libffi/3.3-GCCcore-9.3.0/lib64:/sw/arch/Debian10/EB_production/2020/software/libffi/3.3-GCCcore-9.3.0/lib:/sw/arch/Debian10/EB_production/2020/software/GMP/6.2.0-GCCcore-9.3.0/lib:/sw/arch/Debian10/EB_production/2020/software/SQLite/3.31.1-GCCcore-9.3.0/lib:/sw/arch/Debian10/EB_production/2020/software/Tcl/8.6.10-GCCcore-9.3.0/lib:/sw/arch/Debian10/EB_production/2020/software/libreadline/8.0-GCCcore-9.3.0/lib:/sw/arch/Debian10/EB_production/2020/software/ncurses/6.2-GCCcore-9.3.0/lib:/sw/arch/Debian10/EB_production/2020/software/bzip2/1.0.8-GCCcore-9.3.0/lib:/sw/arch/Debian10/EB_production/2020/software/OpenMPI/4.0.3-GCC-9.3.0/lib:/sw/arch/Debian10/EB_production/2020/software/UCX/1.8.0-GCCcore-9.3.0/lib:/sw/arch/Debian10/EB_production/2020/software/hwloc/2.2.0-GCCcore-9.3.0/lib:/sw/arch/Debian10/EB_production/2020/software/libpciaccess/0.16-GCCcore-9.3.0/lib:/sw/arch/Debian10/EB_production/2020/software/libxml2/2.9.10-GCCcore-9.3.0/lib:/sw/arch/Debian10/EB_production/2020/software/XZ/5.2.5-GCCcore-9.3.0/lib:/sw/arch/Debian10/EB_production/2020/software/numactl/2.0.13-GCCcore-9.3.0/lib:/sw/arch/Debian10/EB_production/2020/software/binutils/2.34-GCCcore-9.3.0/lib:/sw/arch/Debian10/EB_production/2020/software/zlib/1.2.11-GCCcore-9.3.0/lib:/sw/arch/Debian10/EB_production/2020/software/GCCcore/9.3.0/lib64:/sw/arch/Debian10/EB_production/2020/software/GCCcore/9.3.0/lib
2021-02-14 21:38:44.912062: W tensorflow/stream_executor/cuda/cuda_driver.cc:312] failed call to cuInit: UNKNOWN ERROR (303)
2021-02-14 21:38:44.912080: I tensorflow/stream_executor/cuda/cuda_diagnostics.cc:156] kernel driver does not appear to be running on this host (r37n1.lisa.surfsara.nl): /proc/driver/nvidia/version does not exist
2021-02-14 21:38:44.912037: W tensorflow/stream_executor/platform/default/dso_loader.cc:59] Could not load dynamic library 'libcuda.so.1'; dlerror: libcuda.so.1: cannot open shared object file: No such file or directory; LD_LIBRARY_PATH: /sw/arch/Debian10/EB_production/2020/software/OpenMPI/4.0.3-GCC-9.3.0/lib:/sw/arch/Debian10/EB_production/2020/software/OpenMPI/4.0.3-GCC-9.3.0/lib:/sw/arch/Debian10/EB_production/2020/software/Python/3.8.2-GCCcore-9.3.0/lib:/sw/arch/Debian10/EB_production/2020/software/libffi/3.3-GCCcore-9.3.0/lib64:/sw/arch/Debian10/EB_production/2020/software/libffi/3.3-GCCcore-9.3.0/lib:/sw/arch/Debian10/EB_production/2020/software/GMP/6.2.0-GCCcore-9.3.0/lib:/sw/arch/Debian10/EB_production/2020/software/SQLite/3.31.1-GCCcore-9.3.0/lib:/sw/arch/Debian10/EB_production/2020/software/Tcl/8.6.10-GCCcore-9.3.0/lib:/sw/arch/Debian10/EB_production/2020/software/libreadline/8.0-GCCcore-9.3.0/lib:/sw/arch/Debian10/EB_production/2020/software/ncurses/6.2-GCCcore-9.3.0/lib:/sw/arch/Debian10/EB_production/2020/software/bzip2/1.0.8-GCCcore-9.3.0/lib:/sw/arch/Debian10/EB_production/2020/software/OpenMPI/4.0.3-GCC-9.3.0/lib:/sw/arch/Debian10/EB_production/2020/software/UCX/1.8.0-GCCcore-9.3.0/lib:/sw/arch/Debian10/EB_production/2020/software/hwloc/2.2.0-GCCcore-9.3.0/lib:/sw/arch/Debian10/EB_production/2020/software/libpciaccess/0.16-GCCcore-9.3.0/lib:/sw/arch/Debian10/EB_production/2020/software/libxml2/2.9.10-GCCcore-9.3.0/lib:/sw/arch/Debian10/EB_production/2020/software/XZ/5.2.5-GCCcore-9.3.0/lib:/sw/arch/Debian10/EB_production/2020/software/numactl/2.0.13-GCCcore-9.3.0/lib:/sw/arch/Debian10/EB_production/2020/software/binutils/2.34-GCCcore-9.3.0/lib:/sw/arch/Debian10/EB_production/2020/software/zlib/1.2.11-GCCcore-9.3.0/lib:/sw/arch/Debian10/EB_production/2020/software/GCCcore/9.3.0/lib64:/sw/arch/Debian10/EB_production/2020/software/GCCcore/9.3.0/lib
2021-02-14 21:38:44.912062: W tensorflow/stream_executor/cuda/cuda_driver.cc:312] failed call to cuInit: UNKNOWN ERROR (303)
2021-02-14 21:38:44.912078: I tensorflow/stream_executor/cuda/cuda_diagnostics.cc:156] kernel driver does not appear to be running on this host (r37n1.lisa.surfsara.nl): /proc/driver/nvidia/version does not exist
2021-02-14 21:38:44.912298: I tensorflow/core/platform/cpu_feature_guard.cc:142] This TensorFlow binary is optimized with oneAPI Deep Neural Network Library (oneDNN)to use the following CPU instructions in performance-critical operations:  AVX2 AVX512F FMA
To enable them in other operations, rebuild TensorFlow with the appropriate compiler flags.
2021-02-14 21:38:44.912298: I tensorflow/core/platform/cpu_feature_guard.cc:142] This TensorFlow binary is optimized with oneAPI Deep Neural Network Library (oneDNN)to use the following CPU instructions in performance-critical operations:  AVX2 AVX512F FMA
To enable them in other operations, rebuild TensorFlow with the appropriate compiler flags.
2021-02-14 21:38:44.912282: W tensorflow/stream_executor/platform/default/dso_loader.cc:59] Could not load dynamic library 'libcuda.so.1'; dlerror: libcuda.so.1: cannot open shared object file: No such file or directory; LD_LIBRARY_PATH: /sw/arch/Debian10/EB_production/2020/software/OpenMPI/4.0.3-GCC-9.3.0/lib:/sw/arch/Debian10/EB_production/2020/software/OpenMPI/4.0.3-GCC-9.3.0/lib:/sw/arch/Debian10/EB_production/2020/software/Python/3.8.2-GCCcore-9.3.0/lib:/sw/arch/Debian10/EB_production/2020/software/libffi/3.3-GCCcore-9.3.0/lib64:/sw/arch/Debian10/EB_production/2020/software/libffi/3.3-GCCcore-9.3.0/lib:/sw/arch/Debian10/EB_production/2020/software/GMP/6.2.0-GCCcore-9.3.0/lib:/sw/arch/Debian10/EB_production/2020/software/SQLite/3.31.1-GCCcore-9.3.0/lib:/sw/arch/Debian10/EB_production/2020/software/Tcl/8.6.10-GCCcore-9.3.0/lib:/sw/arch/Debian10/EB_production/2020/software/libreadline/8.0-GCCcore-9.3.0/lib:/sw/arch/Debian10/EB_production/2020/software/ncurses/6.2-GCCcore-9.3.0/lib:/sw/arch/Debian10/EB_production/2020/software/bzip2/1.0.8-GCCcore-9.3.0/lib:/sw/arch/Debian10/EB_production/2020/software/OpenMPI/4.0.3-GCC-9.3.0/lib:/sw/arch/Debian10/EB_production/2020/software/UCX/1.8.0-GCCcore-9.3.0/lib:/sw/arch/Debian10/EB_production/2020/software/hwloc/2.2.0-GCCcore-9.3.0/lib:/sw/arch/Debian10/EB_production/2020/software/libpciaccess/0.16-GCCcore-9.3.0/lib:/sw/arch/Debian10/EB_production/2020/software/libxml2/2.9.10-GCCcore-9.3.0/lib:/sw/arch/Debian10/EB_production/2020/software/XZ/5.2.5-GCCcore-9.3.0/lib:/sw/arch/Debian10/EB_production/2020/software/numactl/2.0.13-GCCcore-9.3.0/lib:/sw/arch/Debian10/EB_production/2020/software/binutils/2.34-GCCcore-9.3.0/lib:/sw/arch/Debian10/EB_production/2020/software/zlib/1.2.11-GCCcore-9.3.0/lib:/sw/arch/Debian10/EB_production/2020/software/GCCcore/9.3.0/lib64:/sw/arch/Debian10/EB_production/2020/software/GCCcore/9.3.0/lib
2021-02-14 21:38:44.912298: W tensorflow/stream_executor/cuda/cuda_driver.cc:312] failed call to cuInit: UNKNOWN ERROR (303)
2021-02-14 21:38:44.912309: I tensorflow/stream_executor/cuda/cuda_diagnostics.cc:156] kernel driver does not appear to be running on this host (r37n1.lisa.surfsara.nl): /proc/driver/nvidia/version does not exist
2021-02-14 21:38:44.912514: I tensorflow/core/platform/cpu_feature_guard.cc:142] This TensorFlow binary is optimized with oneAPI Deep Neural Network Library (oneDNN)to use the following CPU instructions in performance-critical operations:  AVX2 AVX512F FMA
To enable them in other operations, rebuild TensorFlow with the appropriate compiler flags.
2021-02-14 21:38:44.912342: I tensorflow/core/platform/cpu_feature_guard.cc:142] This TensorFlow binary is optimized with oneAPI Deep Neural Network Library (oneDNN)to use the following CPU instructions in performance-critical operations:  AVX2 AVX512F FMA
To enable them in other operations, rebuild TensorFlow with the appropriate compiler flags.
2021-02-14 21:38:44.924621: I tensorflow/core/platform/profile_utils/cpu_utils.cc:104] CPU Frequency: 2100000000 Hz
2021-02-14 21:38:44.924976: I tensorflow/compiler/xla/service/service.cc:168] XLA service 0x3c49b30 initialized for platform Host (this does not guarantee that XLA will be used). Devices:
2021-02-14 21:38:44.925003: I tensorflow/compiler/xla/service/service.cc:176]   StreamExecutor device (0): Host, Default Version
2021-02-14 21:38:44.925233: I tensorflow/core/platform/profile_utils/cpu_utils.cc:104] CPU Frequency: 2100000000 Hz
2021-02-14 21:38:44.925669: I tensorflow/compiler/xla/service/service.cc:168] XLA service 0x3d76cd0 initialized for platform Host (this does not guarantee that XLA will be used). Devices:
2021-02-14 21:38:44.925694: I tensorflow/compiler/xla/service/service.cc:176]   StreamExecutor device (0): Host, Default Version
2021-02-14 21:38:44.940616: I tensorflow/core/platform/profile_utils/cpu_utils.cc:104] CPU Frequency: 2100000000 Hz
2021-02-14 21:38:44.940601: I tensorflow/core/platform/profile_utils/cpu_utils.cc:104] CPU Frequency: 2100000000 Hz
2021-02-14 21:38:44.941054: I tensorflow/compiler/xla/service/service.cc:168] XLA service 0x45826f0 initialized for platform Host (this does not guarantee that XLA will be used). Devices:
2021-02-14 21:38:44.941076: I tensorflow/compiler/xla/service/service.cc:176]   StreamExecutor device (0): Host, Default Version
2021-02-14 21:38:44.941116: I tensorflow/compiler/xla/service/service.cc:168] XLA service 0x4dd4c50 initialized for platform Host (this does not guarantee that XLA will be used). Devices:
2021-02-14 21:38:44.941145: I tensorflow/compiler/xla/service/service.cc:176]   StreamExecutor device (0): Host, Default Version

real	5m49.981s
user	43m36.413s
sys	1m17.977s
